nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/ImagingComponent.java

```diff
@@ -26,7 +26,7 @@
 import static org.nuxeo.ecm.platform.picture.api.ImagingConvertConstants.OPTION_RESIZE_HEIGHT;
 import static org.nuxeo.ecm.platform.picture.api.ImagingConvertConstants.OPTION_RESIZE_WIDTH;
 
-import java.awt.Point;
+import java.awt.*;
```

Comment:
Why?

Reply:
Yes please don't do it, we don't use wildcard imports.

Reply:
The problem is from Intellij IDEA, despite the configuration to avoid using the wildcard. for the  two packages below it will do it ;
java.awt and javax.swing. I will ask Thomas to show me how to update the documentation.